_unittest/conftest.py

```diff
@@ -85,12 +85,8 @@ def setup_class(self, project_name=None, design_name=None, solution_type=None, a
                 application = Hfss
 
             self.aedtapp = application(projectname=self.test_project, designname=design_name,
```

Comment:
During this refactor, can we also change the kwargs to `project_name` and `design_name`?  Would be great to be consistent with snake case.  Pinging @jleonatti as well.